pkg/api/api.go

```diff
@@ -147,6 +147,9 @@ func (a *API) RegisterAPI(statusService statusv1.StatusServiceServer) error {
 	a.RegisterRoutesWithPrefix("/ui/assets/", http.StripPrefix("/ui/", http.FileServer(uiAssets)), false, true, "GET")
 	// Serve index to all other pages
 	a.RegisterRoutesWithPrefix("/ui/", uiIndexHandler, false, true, "GET")
+	// Redirect `/ui` to `/ui/`.
+	// See more: https://github.com/grafana/phlare/pull/649#issuecomment-1522958157.
+	a.RegisterRoute("/ui", http.RedirectHandler("/ui/", http.StatusMovedPermanently), false, true, "GET")
```

Comment:
Nit: I think a 302 redirect has less risk, should we choose to do something else in the future

 ```suggestion
	a.RegisterRoute("/ui", http.RedirectHandler("/ui/", http.StatusFound), false, true, "GET")
```